hedera/tutorials/smart-contracts/hss-x-evm-part-1-schedule-smart-contract-calls.mdx

```diff
@@ -0,0 +1,851 @@
+---
+title: "HSS x EVM - Schedule Smart Contract Calls (Part 1)"
+---
+
+On most EVM chains like Ethereum, smart contracts cannot "wake up" on their own—every function call must be triggered by an externally owned account (EOA) or an off-chain bot. This means implementing time-based automation (like cron jobs) requires external infrastructure.
+
+**Hedera changes this fundamentally.**
+
+With the **[Hedera Schedule Service (HSS) via HIP-755](https://hips.hedera.com/hip/hip-755)** and **[HIP-1215](https://hips.hedera.com/hip/hip-1215)**, smart contracts on Hedera can schedule future calls to themselves or other contracts. The Hedera network itself stores and executes these scheduled transactions when the time comes—**no off-chain bots required**.
+
+This unlocks powerful new patterns:
+
+- **On-chain cron jobs** for DeFi rebalancing and automation
+- **Time-based vesting** and token releases
+- **Recurring payments** and subscriptions
+- **DAO governance** with time-delayed execution
+
+In this tutorial, you'll build a simple **AlarmClock** contract that demonstrates this unique capability by scheduling one-shot and recurring alarms entirely on-chain.
+
+<Info>
```

Comment:
Can we use a `<Tip>` (green callout) or `<Note>` (blue callout). A regular `<Info>` callout appears as a white call out which is not very visible/call-out-ish